codebundles/k8s-deployment-healthcheck/runbook.robot

```diff
@@ -16,7 +16,7 @@ Suite Setup         Suite Initialization
 *** Tasks ***
 Check Deployment Log For Issues
     [Documentation]    Fetches recent logs for the given deployment in the namespace and checks the logs output for issues.
-    [Tags]    fetch    log    pod    container    errors    inspect    trace    info    deployment
+    [Tags]    fetch    log    pod    container    errors    inspect    trace    info    deployment    <service_name> get deployment issue logs
```

Comment:
I'm not sure what's happening here with Tags, it feels like it's not ... right...

Reply:
Like... Wouldn't the task name suffice? Or adjust the task name accordingly?

Reply:
My original intent here was to signify what name entity types I want to apply to this task, and where in the string, as there might be additional <args> positions in the future. 

If I rely on using the task name, all name entities may apply to the task name, of all code bundle tasks. Unless we do something like the following:

1. Replace the tags similar to `<service_name> get deployment issue logs` in this PR with `<service_name>`
2. If `<service_name>` is in task tags,  then merge the <service name> with the task name in the platform backend
3. Update all set_issue_next_steps to use task names (eg. ${service_name} Check Deployment Log For Issues)

@stewartshea I think this is more what you are thinking?